lms/templates/ccx/underscore/form.underscore

```diff
@@ -0,0 +1,181 @@
+    <select name="vertical" id="ccx_vertical" disabled></select>
+  <% } %>
+</div>
+
+<div class="ccx_start_date_time_fields">
+  <div class="field datepair">
+    <label for="ccx_start_date" class="form-label">
+      <b><%- gettext('Start Date') %></b>
+      <span class="sr">
+        <%
+          // Translators: This explains to people using a screen reader how to interpret the format of YYYY-MM-DD
+        %>
+        &nbsp;<%- gettext('format four digit year dash two digit month dash two digit day') %>
+      </span>
+    </label>
+    <% if (selectedSubsection) {  %>
+      <% if (selectedSubsection.start) {  %>
+        <input placeholder="<%- gettext('yyyy-mm-dd') %>" value="<%= selectedSubsection.start.split(' ')[0] %>"
+          type="text" class="date" name="start_date" id="ccx_start_date" />
+      <% } else { %>
+         <input placeholder="<%- gettext('yyyy-mm-dd') %>"
+          type="text" class="date" name="start_date" id="ccx_start_date" />
+      <% } %>
+    <% } else if (selectedChapter) {  %>
+      <% if (selectedChapter.start) {  %>
+        <input placeholder="<%- gettext('yyyy-mm-dd') %>" value="<%= selectedChapter.start.split(' ')[0] %>"
+          type="text" class="date" name="start_date" id="ccx_start_date" />
+      <% } else { %>
+         <input placeholder="<%- gettext('yyyy-mm-dd') %>"
+          type="text" class="date" name="start_date" id="ccx_start_date" />
+      <% } %>
+    <% } else { %>
+      <input placeholder="<%- gettext('yyyy-mm-dd') %>"
+        type="text" class="date" name="start_date" id="ccx_start_date" disabled />
+    <% } %>
+
+     <%
+       // Translators: This explains to people using a screen reader how to interpret the format of HH:MM
+     %>
+     <label for="ccx_start_time" class="sr form-label">
+        <%- gettext('Start time format two digit hours colon two digit minutes') %>
+     </label>
+     <% if (selectedSubsection) {  %>
+       <% if (selectedSubsection.start) {  %>
+         <input placeholder="<%- gettext('time') %>" value="<%= selectedSubsection.start.split(' ')[1] %>"
+          type="text" class="time" name="start_time" id="ccx_start_time" />
+       <% } else { %>
+         <input placeholder="<%- gettext('time') %>"
+          type="text" class="time" name="start_time" id="ccx_start_time" />
+      <% } %>
+    <% } else if (selectedChapter) {  %>
+      <% if (selectedChapter.start) {  %>
+        <input placeholder="<%- gettext('time') %>" value="<%= selectedChapter.start.split(' ')[1] %>"
+          type="text" class="time" name="start_time" id="ccx_start_time" />
+      <% } else { %>
+         <input placeholder="<%- gettext('time') %>"
+          type="text" class="time" name="start_time" id="ccx_start_time" />
+      <% } %>
+    <% } else { %>
+      <input placeholder="<%- gettext('time') %>"
+       type="text" class="time" name="start_time" id="ccx_start_time" disabled />
+    <% } %>
+  </div>
+</div>
+
+<div class="ccx_due_date_time_fields">
```

Comment:
To avoid duplicating the Start Date HTML above, pull out a shared template between Start Date and Due Date.